releasenotes/notes/apm-add-default-uds-listener-01abbd2119552312.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Add a default UDS listener on /var/run/datadog/apm.socket.
+    This is used in the Single Step APM Instrumentation, improving the overall onboarding experience and minimizing the agent configuration.
```

Comment:
This should mention changes to dogstatsd too.

Reply:
good catch!